src/dotnet-install.ps1

```diff
@@ -244,6 +244,10 @@ function Get-NormalizedQuality([string]$Quality) {
         return ""
     }
 
+    if (![string]::IsNullOrEmpty($Version)) {
+        throw "Either Quality or Version option has to be specified. See https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#:~:text=valid%20for%20Windows.)-,%2DQuality%7C%2D%2Dquality%20%3CQUALITY%3E,-Downloads%20the%20latest for details."
```

Comment:
This is useful, but I don't think this anchor-syntax is available on all browsers - I believe it is only supported on chromium-based browsers.  Is there a 'normal' HTML anchor on this docs page that can be used instead? If not, can we modify/work with the docs team to add something more linkable?